src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;

```

Comment:
this field must be unique
```suggestion
    @Column(unique = true)
    private String email;

```